Cargo.toml

```diff
@@ -57,6 +57,7 @@ parse-display = "0.8.0"
 parity-scale-codec = "3.1"
 ink = "4.0.0-beta"
 scale-info = "2.3"
+stellar-xdr = { version = "0.0.12", default-features = false, features = ["std", "next", "base64"] }
```

Comment:
Stellar XDR isn't used by contracts and doesn't end up in the WASM code/instructions that get generated, but it is imported because Soroban contracts embed an API specification for the contract functions and custom types, and that is encoded in XDR and embedded in a custom section of the .wasm file.